sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Program.cs

```diff
@@ -0,0 +1,132 @@
+                environment.TryGetValue("EVENTHUB_NAME_CBPT", out testConfiguration.EventHub);
+                environment.TryGetValue("EVENTHUB_PARTITIONS_CBPT", out testConfiguration.NumPartitions);
+                PromptForResources(testConfiguration, "ConcurrentBufferedProducerTest");
+
+                testConfiguration.ConcurrentSends = 5;
+
+                var testRun = new BufferedProducerTest(testConfiguration);
+                await testRun.Run();
+            }
+
+            if (runBurstBufferedProducerTest)
+            {
+                var testConfiguration = new ProducerConfiguration();
+                environment.TryGetValue("EVENTHUB_NAMESPACE_CONNECTION_STRING", out testConfiguration.EventHubsConnectionString);
+                environment.TryGetValue("APPINSIGHTS_INSTRUMENTATIONKEY", out testConfiguration.InstrumentationKey);
+                environment.TryGetValue("EVENTHUB_NAME_BBPT", out testConfiguration.EventHub);
+                environment.TryGetValue("EVENTHUB_PARTITIONS_BBPT", out testConfiguration.NumPartitions);
+                PromptForResources(testConfiguration, "BurstBufferedProducerTest");
+
+                testConfiguration.ConcurrentSends = 5;
+                testConfiguration.ProducerPublishingDelay = TimeSpan.FromMinutes(15);
+
+                var testRun = new BufferedProducerTest(testConfiguration);
+                await testRun.Run();
+            }
+        }
+
+        private static void PromptForResources(ProducerConfiguration testConfiguration, string testName)
+        {
+            // Prompt for the App Insights instrumentation key
+            while (string.IsNullOrEmpty(testConfiguration.InstrumentationKey))
+                {
+                    Console.Write("Please provide the instrumentation key for the App Insights resource: ");
+                    testConfiguration.InstrumentationKey = Console.ReadLine().Trim();
+                }
+
+            // Prompt for the Event Hubs connection string
+            while (string.IsNullOrEmpty(testConfiguration.EventHubsConnectionString))
+            {
+                Console.Write("Please provide the connection string for the Event Hubs namespace that you'd like to use and then press Enter: ");
+                testConfiguration.EventHubsConnectionString = Console.ReadLine().Trim();
+                Console.WriteLine();
+            }
+
+            // Prompt for the Event Hub names, if they weren't passed.
+
+            while (string.IsNullOrEmpty(testConfiguration.EventHub))
+            {
+                Console.Write($"Please provide the name of the Event Hub that you'd like to use for the {testName} and then press Enter: ");
+                testConfiguration.EventHub = Console.ReadLine().Trim();
+                Console.WriteLine();
+            }
+        }
+
+        private static Dictionary<string,bool> ParseArguments(string[] args)
```

Comment:
nifty.   We can probably shift to using something like `System.CommandLine.Dragonfruit` to make arguments typed and avoid having to parse.

Reply:
I have some examples of this [here](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/kubernetes/generator/Stress.Generator/Program.cs#L17) if you're interested.